src/components/NotificationCard.tsx

```diff
@@ -0,0 +1,25 @@
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { format } from 'date-fns';
```

Comment:
lurer på om vi burde bare unwrappe alle som er mulig. vet ikke om det hjelper på performance, men holde oss til en type import måte.
```suggestion
import format from 'date-fns/format';
```